packages/web/src/pages/purchases-and-sales/components/NoTransactionsContent.tsx

```diff
@@ -9,21 +9,21 @@ import { Icon } from 'components/Icon'
 import { Tile } from 'components/tile'
 import { Text } from 'components/typography'
 
-import styles from './NoPurchasesContent.module.css'
+import styles from './NoTransactionsContent.module.css'
 
-export type NoPurchasesContentProps = {
+export type NoTransactionsContentProps = {
   headerText: string
   bodyText: string
   ctaText: string
   onCTAClicked: () => void
 }
 
-export const NoPurchasesContent = ({
+export const NoTransactionsContent = ({
```

Comment:
All that's missing is an `icon` prop and I think this could be a super general `NoRows` or `EmptyTable` component :eyes: